src/embezzler.ts

```diff
@@ -522,25 +522,80 @@ export const embezzlerSources = [
       adventureMacro($location`Cobb's Knob Treasury`, embezzlerMacro());
     }
   ),*/
+  new EmbezzlerFight(
+    "Lucky!",
+    () => canAdv($location`Cobb's Knob Treasury`, true) && have($effect`Lucky!`),
+    () => (canAdv($location`Cobb's Knob Treasury`, true) && have($effect`Lucky!`) ? 1 : 0),
+    () => {
+      adventureMacro($location`Cobb's Knob Treasury`, embezzlerMacro());
+    }
+  ),
+  /*new EmbezzlerFight(
```

Comment:
I know that we currently have commented code in here because of the pillkeeper, but I think we should refrain from actively adding commented code to garbo--going to ask @phulin if he agrees that these are meaningfully different, though

Reply:
yeah, please do not add new commented code